core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/Purchases.kt

```diff
@@ -99,6 +101,13 @@ public expect class Purchases {
             features: List<BillingFeature> = listOf(),
             callback: (Boolean) -> Unit,
         )
+
+        /**
+         * Given a url string, parses the link and returns a [WebPurchaseRedemption], which can
+         * be used to redeem a web purchase using [Purchases.redeemWebPurchase]
+         * @return A parsed version of the link or null if it's not a valid RevenueCat web purchase redemption link.
```

Comment:
Just a suggestion to consistently use `url` instead of `link`, since the parameter is named `url`:

```suggestion
         * Given a url string, parses it and returns a [WebPurchaseRedemption], which can
         * be used to redeem a web purchase using [Purchases.redeemWebPurchase]
         * @return A parsed [WebPurchaseRedemption] or null if it's not a valid RevenueCat web purchase redemption url.
```